src/step_definitions/elements.ts

```diff
@@ -3,9 +3,10 @@ import { When, Then } from 'cucumber';
 import { comparators } from '../comparators';
 import config from '../core/config.helper';
 import { matchers, regexBuilder } from '../matchers';
-import { waitForCondition } from '../web/cucumber/wait-for-condition.helper';
+import { waitForCondition } from '../web/methods/wait-for-condition.methods';
 import variableStore from '../core/variable-store.helper';
 import { createValueToTextTransformer } from '../transformers/transformer/values-to-text.transformer';
+import { methods } from '../web/methods';
 
 const valueToTextTransformer = createValueToTextTransformer();
 const timeout = parseInt(config.elementsVisibilityTimeout) * 1000;
```

Comment:
Timeout is no more used in 2 steps, maybe we should move it just into the ```When(/^I wait for "([^"]*)" of the "([^"]*)" element$/, function(condition, elementName)``` step?